cmd/kosli/root.go

```diff
@@ -421,15 +421,11 @@ func initialize(cmd *cobra.Command, out io.Writer) error {
 	logger.DebugEnabled = global.Debug
 
 	var err error
-	kosliClient, err = requests.NewStandardKosliClient(global.HttpProxy)
+	kosliClient, err = requests.NewKosliClient(global.HttpProxy, global.MaxAPIRetries, global.Debug, logger)
```

Comment:
I believe debug is no longer used/needed anymore on the kosli client as it is now done via the logger